packages/zen-bull-vault/src/UniOracle.sol

```diff
@@ -1,5 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0-or-later
-
+//SPDX-License-Identifier: BUSL-1.1
```

Comment:
not sure this can be BSL, all the uniswap imports are GPL2

Reply:
It also imports our contract ``OracleLibrary.sol`` which is ``// SPDX-License-Identifier: GPL-2.0-or-later
``, but maybe that is ok as its our contract?